kratos/utilities/color_utilities.cpp

```diff
@@ -0,0 +1,42 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// Project includes
+#include "utilities/color_utilities.h"  // Must be included first in order to include define.h
+
+// System includes
+#ifdef KRATOS_COMPILED_IN_WINDOWS
+    #include <stdio.h>
+    #ifndef _WIN32_WINNT
+        #define _WIN32_WINNT 0x0601 // Windows 7 or later. See https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=vs-2019
+    #endif
+    #include <windows.h>
+    #if defined(__MINGW32__) || defined(__MINGW64__)
+        #include <winbase.h>
+    #endif
+#endif
+
+// External includes
+
+namespace Kratos
+{
+namespace ColorUtilities
+{
+void InitializeSTDOutput()
+{
+#ifdef KRATOS_COMPILED_IN_WINDOWS
+    SetConsoleMode(GetStdHandle(STD_OUTPUT_HANDLE), ENABLE_VIRTUAL_TERMINAL_PROCESSING);
```

Comment:
why is this needed?

If this is needed, then I think we should integrate it globally, i.e. when launching the Kernel and not each time you need it (as you do now)

Or am I overseeing sth?

Reply:
I will explain. windows.h is a pain in the ass, it conflict with everything, it is the conflict [lich](https://www.youtube.com/watch?v=FrlymHW0qU8), the only way to content its conflict power is to confine it in a cpp, so it is harmful. This was the only way I can ensure you, if you include it in a header all flags will fail (for some Microsoft  reason). The others ones are for compatibility reasons, specially for MinGW